controllers/storagecluster/provider_server.go

```diff
@@ -354,16 +296,12 @@ func GetProviderAPIServerDeployment(instance *ocsv1.StorageCluster) *appsv1.Depl
 							Command: []string{"/usr/local/bin/provider-api"},
 							Env: []corev1.EnvVar{
 								{
-									Name:  util.WatchNamespaceEnvVar,
-									Value: os.Getenv(util.WatchNamespaceEnvVar),
-								},
-								{
-									Name:  "STORAGE_CLUSTER_NAME",
-									Value: instance.Name,
-								},
-								{
-									Name:  "STORAGE_CLUSTER_UID",
-									Value: string(instance.UID),
```

Comment:
Why are we removing these?

Reply:
These are not used in the server hence removing the unused code